main.tf

```diff
@@ -0,0 +1,62 @@
+module "kvstore_label" {
+  source  = "cloudposse/label/null"
+  version = "0.25.0" # requires Terraform >= 0.13.0
+
+  for_each = var.kvstores
+
+  enabled             = module.this.enabled
+  namespace           = module.this.namespace
+  tenant              = module.this.tenant
+  environment         = module.this.environment
+  stage               = module.this.stage
+  name                = module.this.name
+  delimiter           = module.this.delimiter
+  tags                = module.this.tags
+  additional_tag_map  = module.this.additional_tag_map
```

Comment:
the context.tf is part of the repository prepping, so i don't need to commit it as it exists already. For this label module, i do explicitly not want to have the attributes from the module.this.attributes reference. That is because attributes are always added, never replaced when you pass them in.
Let's imagine this:
```hcl
module "this" {
  source  = "cloudposse/label/null"
  version = "0.25.0"

  attributes          = ["bar"]
}
module "kvstore_label" {
  source  = "cloudposse/label/null"
  version = "0.25.0"

  context             = module.this.context
  attributes          = ["foo"]
}
```
Then my kvstore_label module suddenly has the attributes=["bar", "foo"].
This is unwanted behaviour, because we want `each.key` to be the first or last attribute, depending on var.attributes_as_suffix.